README.md

```diff
@@ -154,6 +154,30 @@ jobs:
       - run: java -cp java HelloWorldApp
 ```
 
+### Install multiple JDKs
+```yaml
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - run: echo "JAVA_8=$JAVA_HOME" >> $GITHUB_ENV
+  
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - run: echo "JAVA_11=$JAVA_HOME" >> $GITHUB_ENV
```

Comment:
```suggestion
      - run: echo "JAVA_11_HOME=$JAVA_HOME" >> $GITHUB_ENV
```

Reply:
The actions manages its own *_HOME variables and i prefer to do not use this postfix in the sample snippet in order to avoid possible clashes. Moreover having "little bit quick and dirty" naming emphases the fact the variables are just temporary holders for internal use